src/test/java/com/netcracker/study/general/GeneralTest.java

```diff
@@ -1,14 +1,23 @@
 package com.netcracker.study.general;
 
-import com.netcracker.study.objects.borders.*;
+import com.netcracker.study.objects.borders.Passage;
+import com.netcracker.study.general.Direction;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public class GeneralTest {
+
     @Test
     public void testCreatePassage(){
```

Comment:
please refactor test name using my **shouldWhen** pattern. I gave example in another comment